transmart-notifications/grails-app/controllers/org/transmartproject/notifications/NotificationsUrlMappings.groovy

```diff
@@ -0,0 +1,12 @@
+package org.transmartproject.notifications
+
+class NotificationsUrlMappings {
+
+    static mappings = {
+        group "/v2", {
+            group "/admin", {
+                "/subscription/notify"(method: 'GET',controller: 'subscriptionMail', action: 'subscriptionNotify')
```

Comment:
You could also surround this with `if (Holders.config.org.transmartproject.notifications.endpointEnabled) {` (and add an entry in the config file) to be able to disable/enable this endpoint entirely, decause it is supposed to be for testing only.

Reply:
I tried this way, but adding this condition here was causing some issues:
```
Field urlMappings in org.grails.web.mapping.servlet.UrlMappingsErrorPageCustomizer required a bean of type 'grails.web.mapping.UrlMappings' that could not be found.
```
I disabled it in the controller.